content/reference/endpoints/rentals.md

```diff
@@ -170,6 +170,43 @@ GET /rentals
 
 <%= render 'json_response', endpoint: "rentals", scopes: %w(public rentals_read-rentals_write) %>
 
+## Search rentals
+
+Search parameters allow to filter rentals by specified fields.
+
+Example:
+
+~~~
+GET /rentals/search?start_at=20140324&end_at=20140330
+~~~
+
+### Search Parameters
+
+Name              | Type    | Default | Description
+------------------|---------|--------------
+amenities         | String  |         | Rentals with given amenities, each amenity has to be a separate parameter.
+bb_ne             | Float   |         | Rentals located between `bb_sw` & `bb_ne`. Format `latitude, longitude`.
+bb_sw             | Float   |         | Rentals located between `bb_sw` & `bb_ne`. Format `latitude, longitude`.
+city              | String  |         | Rentals matching given city.
+country_code      | String  |         | Rentals matching given country code, list of codes is available at [wikipedia](http://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements).
+destination       | String  |         | List of comma separated destinations.
+direction         | String  |         | Direction to sort rentals, either `asc` or `desc`.
+guests            | Integer |         | Rentals with maximum number of sleeps above given number of guests.
+include_tentative | Boolean | false   | Treat tentative periods as available ones.
+max_price         | Integer |         | Rentals with `max_price` below given price. Will use `initial_price` if computed.
```

Comment:
Currently yes, `initial_price` is already the `final_price` We'll change this and include traces about how the price was computed.